.github/quarkus-github-lottery.yml

```diff
@@ -0,0 +1,38 @@
+notifications:
+  createIssues:
+    repository: "quarkusio/quarkus-github-lottery-reports"
+buckets:
+  triage:
+    # TODO: No such label on quarkusio... ? This won't work without a label.
+    label: "triage/needs-triage"
+    delay: PT0S
+    timeout: P3D
+  # TODO: No area labels on quarkusio... ? Notification to maintainers won't work without area labels.
+  maintenance:
+    feedback:
+      # TODO: No such label on quarkusio... ? This won't work without a label.
+      labels: ["triage/needs-reproducer", "triage/needs-feedback"]
+      needed:
+        delay: P21D
+        timeout: P3D
+      provided:
+        delay: P7D
+        timeout: P3D
+    stale:
+      delay: P21D
+      timeout: P3D
+  # This can work without any labels.
+  stewardship:
+    delay: P21D
+    timeout: P3D
```

Comment:
I picked a shorter delay/timeout than on quarkusio/quarkus, because we have no maintainers on this repo. Was 60d/14d on quarkusio/quarkus.

See: 

* https://github.com/quarkusio/quarkus-github-lottery?tab=readme-ov-file#maintenance
* https://github.com/quarkusio/quarkus-github-lottery?tab=readme-ov-file#stewardship